script/WireAdapters.s.sol

```diff
@@ -17,8 +17,6 @@ import {ILayerZeroAdapter} from "../src/adapters/interfaces/ILayerZeroAdapter.so
 ///
 ///      Intended for testnet use only.
 contract WireAdapters is Script {
-    IAdapter[] adapters; // Storage array like in CommonDeployer
```

Comment:
Q: Why this was removed? I think this worked well given it can be increased by `push()`